contracts/EndaomentAdmin.sol

```diff
@@ -20,18 +20,11 @@ contract TwoStepOwnable {
   address private _owner;
   address private _newPotentialOwner;
```

Comment:
A comment on the audit in item N04 discusses line 85 about _newPotentialOwner now being a cancelled owner, recommending a change for readability. We think that the method describes the function and in its current state is clear.